src/borg/archiver/help_cmd.py

```diff
@@ -180,6 +178,25 @@ class HelpMixIn:
         exclude stops examination of subdirectories so that potential includes
         will not match - use normal excludes for such use cases.
 
+        Example::
+
+            # Define the root path prefix
+            R /
+            # Exlude all iso files in any directory
+            - **/*.iso
+            # Explicitly include some directories and all their content
+            + etc/**
+            + root/**
+            # Exclude specific directory under /home
+            - home/*/.cache
+            # Explicilty include all other directories in /home
+            + home/**
+            # Explicitly exclude some directories without recursion
+            ! re:^(dev|proc|run|sys|tmp)
+            # Exclude all other files and directories
+            # that are not specifically included earlier
+            - **
+
```

Comment:
If you add an example, please make sure it does what the comments say.
Also, read it again, typos.

Reply:
That's the config I use here, and it seems to work. Do you notice something which could not work as described ?

Reply:
Typo done

Reply:
discussed in the latest fixup